target/riscv/csr.c

```diff
@@ -1333,12 +1333,8 @@ static int write_pmpaddr(CPURISCVState *env, int csrno, target_ulong val)
 #endif
 
 #ifdef TARGET_CHERI
-
 // See Capability Control and Status Registers (CCSRs) in CHERI ISA spec
```

Comment:
Hm, either this comment should be grouped with the one below (which should also not have a blank line after it) or it should move to where XCCSR_* are defined

Reply:
I've moved it to the `XCCSR_*` definitions and pruned the second comment.